source/mir/ndslice/topology.d

```diff
@@ -3067,3 +3068,784 @@ unittest
     assert(lowerData == [0, 3, 4, 6, 7, 8]);
     assert(upperData == [0, 1, 2, 4, 5, 8]);
 }
+
+/// Transposed adjusted to ignore dim=0 and include universal
+@fastmath private template adjTransposed(Dimensions...)
+    if (Dimensions.length)
+{
+    import mir.ndslice.slice : Slice, SliceKind;
+    import mir.ndslice.internal : isSize_t, toSize_t;
+    import std.meta : allSatisfy, staticMap;
+   
+    static if (!allSatisfy!(isSize_t, Dimensions))
+        alias adjTransposed = .adjTransposed!(staticMap!(toSize_t, Dimensions));
+    else
+    ///
+    auto adjTransposed(SliceKind kind, size_t[] packs, Iterator)(Slice!(kind, packs, Iterator) slice)
+    {
+        import mir.ndslice.topology : ipack;
+        import mir.ndslice.internal : DimensionsCountCTError, DimensionCTError;
+       
+        mixin DimensionsCountCTError;
+            
+        static if (Dimensions.length == 1 && Dimensions[0] == 0)
```

Comment:
`static if ([Dimensions] == [Iota!(Dimensions.length)])`
or 
`static if (Dimensions == Iota!(Dimensions.length))` but not sure if the last variant works